src/libs/ajax.js

```diff
@@ -517,11 +496,6 @@ export const Election = {
     return res.json();
   },
 
-  findElectionById: async (electionId) => {
-    const url = `${await Config.getApiUrl()}/api/election/${electionId}`;
```

Comment:
unused

Reply:
This is a super handy method to keep in Consent, so let's not deprecate that endpoint.